### PR TITLE
change priorty of reboot queue cancel

### PR DESCRIPTION
**Pull request by YZ775**
In the current CKE, the dequeue operation of cancelled reboot queue entry has lower priority than other operations like drain, reboot, ...etc.
This implementation causes the dequeue operation not to work correctly when we cancel an entry whose node is under operation.
This PR solves the problem by following changes.
- separating dequeue operation between cancelled entry and completed entry
- set dequeue operation of cancelled entry higher

### CODE REVIEW

File: server/control.go [hunk 1]
```diff
@@ -346,13 +346,15 @@ func (c Controller) runOnce(ctx context.Context, leaderKey string, tick <-chan t
 	newlyDrained := op.ChooseDrainedNodes(cluster, apiServers, rqEntries)
 	drainCompleted, drainTimedout, _ := op.CheckDrainCompletion(ctx, inf, nf.HealthyAPIServer(), cluster, rqEntries)
 	rebootDequeued := op.CheckRebootDequeue(ctx, cluster, rqEntries)
+	RebootCancelled := op.CheckRebootCancelled(ctx, cluster, rqEntries)
```
Comment (masa213f) [inline server/control.go:349]: Please use lowercase.

```suggestion
	rebootCancelled := op.CheckRebootCancelled(ctx, cluster, rqEntries)
```

File: server/strategy.go [hunk 2]
```diff
@@ -883,6 +884,11 @@ func rebootOps(c *cke.Cluster, constraints *cke.Constraints, rebootArgs DecideOp
 		return nil, false
 	}
 
+	if len(rebootArgs.RebootCancelled) > 0 {
+		phaseReboot = true
+		ops = append(ops, op.RebootDequeueOp(rebootArgs.RebootCancelled))
```
Comment (masa213f) [inline server/strategy.go:889]: I want to change the display name of the op to `reboot-cancel.`
Is it possible?

**Review by masa213f — CHANGES_REQUESTED** [review]

File: server/strategy.go (hunk 2 above)
Comment (YZ775) [reply server/strategy.go:889]: We can change the display name by duplicating `RebootDequeueOp` and  changing `Name()` method.
The logic of Op is identical. so I created `RebootCancelOp` which has embedded `RebootDequeueOp` and override `Name()` method.

File: op/reboot.go [hunk 3]
```diff
@@ -523,6 +523,25 @@ func (c rebootDequeueCommand) Command() cke.Command {
 	}
 }
 
+//
```
[the remark this answers is not in the document]
Comment (YZ775) [reply op/reboot.go:526]: https://github.com/cybozu-go/cke/blob/7d544a49f6821fab2911b47101c4204b427f8d20/op/reboot.go#L184
https://github.com/cybozu-go/cke/blob/7d544a49f6821fab2911b47101c4204b427f8d20/op/reboot.go#L331

There are `//` between code block of Op, so I added it same as other Op.

[the remark this answers is not in the document]
Comment (masa213f) [reply op/reboot.go:526]: Oh, I see.

**Review by masa213f — APPROVED** [review]
LGTM

**Pull request merged**